includes/SiteGen/SiteGen.php

```diff
@@ -154,6 +155,38 @@ private static function get_patterns_for_category( $category, $site_classificati
 		return $processed_patterns;
 	}
 
+	/**
+	 * Get the templates for a particular category.
+	 *
+	 * @param string $category The category to get templates for.
+	 */
+	private static function get_templates_for_category( $category ) {
```

Comment:
Should we add support for `primary_type` and `secondary_type` as well?

Reply:
I had thought of adding it but did not see much difference after adding it. We can add the support and keep it so that in future when the templates endpoint starts using the primary and secondary type the content will automatically start appearing better.
for eg: 
https://patterns.hiive.cloud/templates?category=contact
https://patterns.hiive.cloud/templates?category=contact&primary_type=food-beverage&secondary_type=coffee-tea
These endpoints show the same content.
Adding the support for now as it might be helpful in the future.